examples/error_unhandling.ts

```diff
@@ -1,14 +1,16 @@
 import { alice, bob } from "capi"
 import { Balances } from "westend_dev/mod.ts"
+import { assertRejects } from "../deps/std/testing/asserts.ts"
 
-await Balances
-  .transfer({
-    value: 1_000_000_000_000_000_000_000_000_000_000_000_000n,
-    dest: bob.address,
-  })
-  .signed({ sender: alice })
-  .sent()
-  .dbgStatus()
-  .finalizedEvents()
-  .unhandleFailed()
-  .run()
+assertRejects(() =>
+  Balances
+    .transfer({
+      value: 1_000_000_000_000_000_000_000_000_000_000_000_000n,
+      dest: bob.address,
+    })
+    .signed({ sender: alice })
+    .sent()
+    .dbgStatus()
+    .finalizedEvents()
+    .unhandleFailed()
+    .run(), "unhandleFailed")
```

Comment:
I don't think the `"unhandleFailed"` here is right.

Reply:
```suggestion
    .run())
```

Reply:
mmm gimme a suggestion. not sure how these should be worded. "should undandleFailed` ?

Reply:
oh just remove it ok

Reply:
Ohh, sorry, I misunderstood that parameter. In that case, perhaps `"Should fail due to `InsufficientBalance`"`?